CMSgov/onemac
# Oy2 19634 - base waiver id format for package view

Story: https://qmacbis.atlassian.net/browse/OY2-19634
Endpoint: https://d1es38x71h06ou.cloudfront.net/

### Details

Update base waiver id required format for package view

### Changes

- Updated baseWaiver and BaseWaiverForm config for format and FAQ link

### Implementation Notes

- Tested to ensure TE on base waiver still functions/links property; Renewal/Waiver not yet implemented.

### Test Plan

1. Login as state submitter
2. Navigate to package list and click New Submisssion and select Waiver Action -> Base Waiver
3. Enter a waiver authority, proposed effective date, and add any attachment
4. Enter a waiver id that already exists and verify red error message (form is not submittable)
5. Enter a waiver id in the incorrect format and verify red error message (form is not submittable)
6. Enter a waiver id in the correct format and submit
